src/Entity/OgMembership.php

```diff
@@ -191,10 +191,7 @@ public function revokeRole(OgRole $role) {
    * {@inheritdoc}
    */
   public function getRoles() {
-    // Add the member role.
```

Comment:
we need this for the getting the permissions. So, maybe another option, is making sure in OgMembership::preSave to strip out the auth role?